### PR TITLE
Better DoltHub landing page

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: content/products/dolthub/README.md [hunk 1]
```diff
@@ -4,4 +4,24 @@ title: DoltHub
...
\ No newline at end of file
+[DoltHub](https://www.dolthub.com) is a place to share Dolt databases. We host public data for free! DoltHub adds a modern, secure, always on database management web GUI to the Dolt ecosystem. Edit your database on the web, have another person review it via a pull request, and have the production database pull it to deploy.
+
+# What is DoltHub
+
+DoltHub is Dolt's GitHub. DoltHub acts as a [Dolt remote](../../concepts/dolt/git/remotes.md) you can [clone, push, pull and fetch](../../reference/cli.md) from. DoltHub adds [permissions](../../concepts/dolthub/permissions.md), [pull requests](../../concepts/dolthub/prs.md), [issues](../../concepts/dolthub/issues.md), and [forks](../../concepts/dolthub/forks.md) to the Dolt ecosystem. Additionally, DoltHub has a modern SQL workbench built in so you can explore and change databases on the web.
```
Comment (tbantle22) [inline content/products/dolthub/README.md]: "DoltHub is GitHub for Dolt" might read a little better

File: content/products/dolthub/README.md [hunk 2]
```diff
@@ -4,4 +4,24 @@ title: DoltHub
...
+
+# DoltHub API
+
+DoltHub has [an API](./dolthub-api/README.md) you can script against. 
+
+# Guides
+
+- [Transform File Uploads](./transform-uploads.md)
+
+We added a feature to [Transform File Uploads](./transform-uploads.md) to DoltHub. This allows you to set up an API to receive and transform files that are uploaded to your databases on DoltHub. The transformed files are sent back to DoltHub for import to a DoltHub database.
```
Comment (tbantle22) [inline content/products/dolthub/README.md]: transform file uploads on DoltHub?

File: content/products/dolthub/README.md [hunk 3]
```diff
@@ -4,4 +4,24 @@ title: DoltHub
...
\ No newline at end of file
+[DoltHub](https://www.dolthub.com) is a place to share Dolt databases. We host public data for free! DoltHub adds a modern, secure, always on database management web GUI to the Dolt ecosystem. Edit your database on the web, have another person review it via a pull request, and have the production database pull it to deploy.
+
+# What is DoltHub
+
+DoltHub is Dolt's GitHub. DoltHub acts as a [Dolt remote](../../concepts/dolt/git/remotes.md) you can [clone, push, pull and fetch](../../reference/cli.md) from. DoltHub adds [permissions](../../concepts/dolthub/permissions.md), [pull requests](../../concepts/dolthub/prs.md), [issues](../../concepts/dolthub/issues.md), and [forks](../../concepts/dolthub/forks.md) to the Dolt ecosystem. Additionally, DoltHub has a modern SQL workbench built in so you can explore and change databases on the web.
+
+# Use Cases
+
+DoltHub can be used for [Data Sharing](./data-sharing.md) publicly or privately.
```
Comment (tbantle22) [inline content/products/dolthub/README.md]: Feels like this and the DoltHub API sections could be elaborated on a little more. Like one more sentence each explaining it

**Review by tbantle22 — APPROVED** [review]

File: content/products/dolthub/README.md (hunk 3 above)
Comment (timsehn) [reply content/products/dolthub/README.md]: Damn u and ur high standards. 

But ur right

**Pull request merged**